acceptance/tests/language/invoke_pp_function_from_nondependent_module.rb

```diff
@@ -0,0 +1,58 @@
+test_name 'Puppet-language functions can be called from inside an another module' do
+
+  mini_func = 'function jenny::mini($a, $b) {if $a <= $b {$a} else {$b}}'
+  maxi_func = 'function jenny::nested::maxi($a, $b) {if $a >= $b {$a} else {$b}}'
+  
+  step 'Create PP functions on master' do
+    manifest = <<-EOM.gsub(/^ {6}/,'')
+      File {
+        ensure => present,
+        owner => 'puppet',
+        group => 'puppet',
+        mode => '0644',
+      }
+      file {['/etc/puppetlabs',
+        '/etc/puppetlabs/code',
+        '/etc/puppetlabs/code/modules/',
+        '/etc/puppetlabs/code/modules/jenny',
+        '/etc/puppetlabs/code/modules/jenny/functions',
+        '/etc/puppetlabs/code/modules/jenny/functions/nested',
+        '/etc/puppetlabs/code/modules/tommy',
+        '/etc/puppetlabs/code/modules/tommy/manifests',
+        '/etc/puppetlabs/code/modules/tutone',
+        '/etc/puppetlabs/code/modules/tutone/manifests']:
+        ensure => directory,
+        mode => '0755',
+      }      
+      file {'/etc/puppetlabs/code/modules/jenny/functions/mini.pp':
+        content => '#{mini_func}',
+      }
+      file {'/etc/puppetlabs/code/modules/jenny/functions/nested/maxi.pp':
+        content => '#{maxi_func}',
+      }
+      file {'/etc/puppetlabs/code/modules/tommy/manifests/init.pp':
+        content => 'class tommy { notify { "tommy says FOO": } }',
+      }
+      file {'/etc/puppetlabs/code/modules/tutone/manifests/testmini.pp':
```

Comment:
@hlindberg I don't think I'm getting the setup correct to call a puppet function from a module that has dependencies, but not on the module defining the function. I expect lines 36-41 to fail, but they succeed.